src/python/turicreate/toolkits/activity_classifier/_tf_model_architecture.py

```diff
@@ -67,7 +68,7 @@ def __init__(self, net_params, batch_size, num_features, num_classes, prediction
         conv = _tf.nn.bias_add(conv, self.biases['conv_bias'])
         conv = _tf.nn.relu(conv)
 
-        dropout = _tf.layers.dropout(conv, rate=0.2, training=self.is_training)
+        dropout = _tf.layers.dropout(conv, rate=0.2, training=self.is_training, seed=seed)
```

Comment:
To future-proof this, could we just do this at the graph level?

Reply:
Actually I think there we got the same problem as OD.
`tf.set_random_seed` doesn't work here either ;(

Reply:
Yeah he is right. Somehow only operation level seed works for dropout!

Reply:
haha seems like this is a known issue?
https://stackoverflow.com/questions/38469632/tensorflow-non-repeatable-results/40247201